README.md

```diff
@@ -1,181 +1,53 @@
-61405.849163   e89fcd33936c     4026532402   4026532405   0      cap_capable      ls               6      6      1      0            CAP_SYS_ADMIN
-61405.849178   e89fcd33936c     4026532402   4026532405   0      lstat            ls               6      6      1      0            ./media
-61405.849201   e89fcd33936c     4026532402   4026532405   0      getdents64       ls               6      6      1      0            3
-61405.849220   e89fcd33936c     4026532402   4026532405   0      close            ls               6      6      1      0            3
-61405.849280   e89fcd33936c     4026532402   4026532405   0      ioctl            ls               6      6      1      0            1 21523
-61405.849318   e89fcd33936c     4026532402   4026532405   0      do_exit          ls               6      6      1      0
-```
-
-As can be seen in the above output, each event line shows the following information about the event:
-
-* TIME - shows the event time relative to system boot time in seconds
-* UTS_NAME - uts namespace name. As there is no container id object in the kernel, and docker/k8s will usually set this to the container id, we use this field to distinguish between containers.
-* MNT_NS - mount namespace inode number.
-* PID_NS - pid namespace inode number. In order to know if there are different containers in the same pid namespace (e.g. in a k8s pod), it is possible to check this value
-* UID - real user id (in host user namespace) of the calling process
-* EVENT - identifies the event (e.g. syscall name)
-* COMM - name of the calling process
-* PID - pid of the calling process
-* TID - tid of the calling thread
-* PPID - parent pid of the calling process
-* RET - value returned by the function
-* ARGS - list of arguments given
-
-Note about string arguments and userspace pointers: as pointers are being dereferenced from userspace memory, a malicious program may change the content being read before it actually gets executed in the kernel. Take this into account when doing security related stuff.
-
-Tracee currently supports a subset of system calls events, which can be listed with:
-
-`./start.py -l`
-
-Other supported events are (functions called in kernel space):
 
-* cap_capable - indicates which capabilities were requested
-* do_exit - indicates exited processes
+This will run Tracee with no arguments which will collect all events from all newly created processes and print them as a table to the standard output.
 
+### Understanding the output
 
-Adding new events (especially system calls) to Tracee is straightforward, but one should keep in mind that tracing too many events may cause system performance degradation. Other than that, high event rate can cause samples to be lost (an error message will then be shown as part of the output). For this reason, *read* and *write* syscalls are deliberately excluded from Tracee.
```

Comment:
You removed the part that says that adding new events to Tracee is easy. I think this is a differentiator from other projects, and it should be noted in the readme

Reply:
@lizrice  suggested this belongs in the contribution guidelines. wdyt?